src/components/TimePicker/index.js

```diff
@@ -136,7 +137,7 @@ class TimePicker extends Component {
         return (
             <StyledContainer id={id} className={className} style={style}>
                 <Input
-                    id="time-picker_time-input"
+                    id={uniqueId('time-picker_time-input')}
```

Comment:
unique Ids are being use wrong here because every time the component renders it will have diff id, they should unique by instance ... I think it should be used as state or using memo

Reply:
this approach is being used everywhere in the implementation